nc-games/src/App.css

```diff
@@ -32,3 +32,24 @@
   width: 15rem;
   height: auto;
 }
+
+/* Comments */
+
+.hideComments {
+  position: absolute;
+  left: -2000px;
+}
+
+.showComments ul {
+  list-style-type: none;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+.showComments ul li {
+  border: 1px solid black;
+  display: flex;
+  width: 15rem;
```

Comment:
It might be nice to make the comment width slightly longer for those long comments so they dont take up half the screen :)